tables/fr-bfu-comp68.cti

```diff
@@ -21,56 +21,56 @@
 
 # Références :
 # - Code braille français uniformisé pour la transcription des textes imprimés (CBFU) - Deuxième édition Septembre 2008
-# - Notation mathématique braille - Première édition janvier 2007
+# - Notation noback mathématique braille - Première édition janvier 2007
```

Comment:
This probably got in here by mistake. I can fix it when merging.

Reply:
Exact. This happened during a *search & replace*. Thanks.